documentation/src/mkdocs.yml

```diff
@@ -70,6 +114,7 @@ pages:
       - 'Pravega Security Authorization/Authentication': 'security/pravega-security-authorization-authentication.md'
       - 'Pravega Security Configurations': 'security/pravega-security-configurations.md'
       - 'Pravega Security Encryption': 'security/pravega-security-encryption.md'
+      - 'Securing Pravega in Standalone Mode': 'security/securing-standalone-mode-cluster.md'
```

Comment:
Is this section including the Keycloak repo? https://github.com/pravega/pravega-keycloak

Reply:
That would be nice, but that's a gap we have so far. I have looked into how to connect pravega-keycloak to FreeIPA, and I think I could easily do it, but I need time to work through the steps and provide a tutorial. I don't know if anyone else has made other progress towards using pravega-keycloak with open source Pravega, or if we have docs on it.